plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/command/RemoveConnectionHandler.java

```diff
@@ -41,7 +44,7 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
 									@Override
 									public String toString(IConnection connection) {
-										return "\n" + connection.toString();
+										return connection == null ? "\nnull" : ("\n" + connection.toString());
```

Comment:
It is now assumed that the connection list is valid (based on the changes in UIUtils), but I simply add the 'null' as a safety to prevent NPE.